exercises/elm-2-a/src/AggregateScorers.elm

```diff
@@ -0,0 +1,5 @@
+module AggregateScorers exposing (..)
```

Comment:
This file can be removed, as the start point for the second phase will be predefined (see earlier comment).